airflow/models/taskinstance.py

```diff
@@ -1693,6 +1693,8 @@ def _execute_task(self, context, task_orig):
             execute_callable = getattr(task_to_execute, self.next_method)
             if self.next_kwargs:
                 execute_callable = partial(execute_callable, **self.next_kwargs)
+        elif self.next_kwargs is not None:
+            raise AirflowException("Task is coming out of deferral without next_method specified.")
```

Comment:
This should use a exception subclass instead

Reply:
the reason I added this is because i was checking out the "debuggability" when user misconfigures this.  i.e. when user uses a trigger that does _not_ exit and user provides no method.

without adding this catch, the task will just run the execute method (which perhaps surprisingly results in an infinite loop of execute and deferral and back again).

but i think i will change this to make the intention clearer